packages/editor-ui/src/components/ExternalSecretsProviderImage.ee.vue

```diff
@@ -1,10 +1,16 @@
 <script lang="ts" setup>
+/* eslint-disable @typescript-eslint/ban-ts-comment */
 import type { PropType } from 'vue';
 import type { ExternalSecretsProvider } from '@/Interface';
 import { computed } from 'vue';
+
+// @ts-expect-error
 import infisical from '../assets/images/infisical.webp';
```

Comment:
you can put a module definition to one of the shim.d.ts files
`declare module '*.webp'`

Reply:
I tried, it doesn't get picked up by `vue-tsc`. 🥲